### DISCUSSION TITLE
[SO 16.04 - Logstash] Problem with tagging for specific IP subnets

### DISCUSSION THREAD

**Discussion by Angelozinna96** [Security-Onion-Solutions/securityonion#2600] [Unsupported Versions] [answered]
Hi everyone,
I'm a newbie here, so please forgive me if I say something obvious or I forget something important.
I'm currently working with SO 16.04, with default configurations, trying to add a tag for each specified subnet using Logstash.
The file .conf that I've created follow this pattern (unique difference with the original .conf file is that I added like 50 cidr for each if statement because I have a lot of subnet to assign):
`filter{
	if[source_ip]{
	cidr{
		address => [ "%{source_ip}" ]
		network => ["192.168.1.0/24"]
		add_tag => ["src_TEST"]
              }
         }
	if[destination_ip]{
	cidr{
		address => [ "%{destination_ip}" ]
		network => ["192.168.1.0/24"]
		add_tag => ["dst_TEST"]
	     }
        }
        if[syslog-sourceip]{
	        cidr{
		        address => [ "%{syslog-sourceip}"]
		        network => ["192.168.1.0/24"]
		        add_tag => ["syslogsrc_TEST"]
                      }
                 }
}
`
The steps that I follow for deplying a new configuration file are:

1. Place the .conf file into /etc/logstash/custom/ 
2. Running sudo so-logstash-restart and sudo so-nids-restart
3. Check if the .conf file is been automatically moved into  /etc/logstash/conf.d/  ,and if not I manually do it and I run the step 2 again

I have tested the same configuration into 2 different SO 16.04 instances (testing and deploying) with the following results:
-TESTING instance: Everything works fine, I can easily found all the src_*, dst_* and syslogsrc_* tags associated with my subnets into Kibana.
-DEPLOYING  instance: I can only see syslogsrc_* tags associated with my subnets (so no scr_* and dst_* tags).
Looking into the "logstash-*" Index pattern into Kibana for the specified fields, all of them have the IP type, but meanwhile the syslog-source have no format specified, both source_ip and destination_ip have the url format.
I am attaching the respective results of "GET _template/logstash" of the two instances.
[deploy_logstash_template.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5801735/deploy_logstash_template.txt)
[test_logstash_template.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5801736/test_logstash_template.txt)

Hope someone is still working with some of the oldest versions of SO or still have willingness to answer to it :D
Angelo

**Comment by weslambert** [comment, marked as answer]
@Angelozinna96, this Discussions group is for Security Onion 2.  Security Onion 16.04 will soon be EOL, and all questions for it should go to https://groups.google.com/group/security-onion.
